### DISCUSSION TITLE
Add a #!channelblacklist command

### DISCUSSION THREAD

**Discussion by Willard21** [DiscordDungeons/Bugs#1130] [Ideas]
Instead of hard-coding that only Mackan can use commands in tavern and basement, it would be a good opportunity to add a server-admin command that blacklists the bot for non-admins in that channel.

**Comment by FelixArgyle** [comment]
this has been a wanted thing forever, tho not just "channel blacklist" more so command blacklist
only allowing "sides" in outside channel etc, party commands in party, pvp in pvp (and general commands too like stats, heal and all general type should be allowed)
hmm could it possibly be setup in the same way how setperm works, maybe tho actually allow the "code input" since i don't think setperm allows that, would be easier going to the site and doing something like
`#!channelblacklist #channel-name code `

**Comment by FelixArgyle** [comment]
Discord added support to bots to disallow commands run in certain channels
